develop/interoperability/intro-to-xcm.md

```diff
@@ -1,4 +1,110 @@
+---
+# Introduction to XCM
+
+## What is 'XCM' - Cross-Consensus Messaging?
+
+XCM (Cross Consensus Messaging) is a language that enables communication between chains in the Polkadot ecosystem. XCM allows blockchains to seamlessly send messages to each other, enabling the transfer of assets, data, and other information across different chains and smart contracts. XCM was devleoped within the Polkadot ecosystem, but is designed to be general enough to provide a common format for cross-consensus communication that can be used anywhere. This document will provide you with an introduction to XCM, covering the basics to help you futher understand the language.
+
+One of Polkadot's main functionalities is interoperability amongst parachains and any other participating consensus-driven systems. XCM is the language through which complex, cross-consensus interactions can occur. Two blockchains can "speak" XCM to seamlessly interact with each other using a standard messaging format.
+
+## A format, not a protocol
+It's important to remember that XCM is a messaging format, and not a protocol. It cannot be used to 'send messages' but instead express what should be done by the receiver. XCM does not define how messages are delivered but rather define how they should look, act, and contain relative instructions to the on-chain actions the message intends to perform.
+
+XCM messagges, by themselves, are not considered transactions. XCM just describes how to change the state of the target network, but the message by itself doesn't perform the state change. 
+
+This partly ties to what is called asynchronous composability, which allows XCM messages to bypass the concept of time-constrained mechanisms, like on-chain scheduling and execution over time in the correct order in which it was intended.
+
+XCM works similarly to how RESTful services use REST as an architectural style of development, where HTTP requests contain specific parameters to perform an action.
+
+## The Four Principles of XCM
+XCM has four high-level core design principles which it stands to follow:
+
+* **Asynchronous**
+
+    XCM messages in no way assume that the sender will be blocking on its completion.
+
+* **Absolute**
+
+    XCM messages are guaranteed to be delivered and interpreted accurately, in order and in a timely fashion. Once a message is sent, one can be sure it will be processed as it was intended to be.
+
+* **Asymmetric**
+
+    XCM messages, by default, do not have results that let the sender know that the message was received - they follow the 'fire and forget' paradigm. Any results must be separately communicated to the sender with an additional message back to the origin.
+
+* **Agnostic**
+
+    XCM makes no assumptions about the nature of the consensus systems between which the messages are being passed. XCM as a message format should be usable in any system that derives finality through consensus.
```

Comment:
```suggestion
## The Four Principles of XCM

XCM has four high-level core design principles which it stands to follow:

- **Asynchronous** - XCM messages do not assume the sender will be blocking on its completion
- **Absolute** - XCM messages are guaranteed to be delivered and interpreted accurately, in an order, and timely fashion. Once a message is sent, one can be sure it will be processed as intended
- **Asymmetric** - XCM messages, by default, do not provide results that inform the sender that the message was received; they follow the 'fire and forget' paradigm. Any results must be communicated separately to the sender with an additional message back to the origin
- **Agnostic** - XCM makes no assumptions about the nature of the consensus systems between which the messages are passed. XCM, as a message format, should be usable in any system that derives finality through consensus
```